src/core/lib/channel/call_tracer.h

```diff
@@ -33,6 +33,18 @@
 #include "src/core/lib/transport/metadata_batch.h"
 #include "src/core/lib/transport/transport.h"
 
+// TODO(yashykt): This might not be the right place for this channel arg, but we
```

Comment:
I think this can live in src/cpp/ext/filters/census/open_census_call_tracer.h, since it's really an OpenCensus-specific channel arg.

Reply:
I think in the future when we have more of a common observability API, this would arg would eventually go there, but right now, I don't have strong opinions one way or the other. Moved to open_census_call_tracer.h